index.html

```diff
@@ -7,173 +7,352 @@
     <link rel="stylesheet" href="styles.css">
   </head>
 
```

Comment:
On the Pull request description, please write a detailed accessibility report about all the checks you made on your website. 
You should have something similar to this:
```
Accessibility Report:

- Page Title - no issue found
- Image text alternatives - Images alt text were modified 
- Text Heading - There was no h1 on the page: This has been corrected.
- Color contrast - No issues found
- Resize - Converted px values to rem for text size accessibility reasons
- Interaction - No issue found
- Moving content - Disabled moving content from the page
- Multimedia - No issues found
- The basic structure of the page - No issues found

```